### PR TITLE
Release 10.3-GA

### PR DESCRIPTION
**Pull request by brb-nv**
## 10.3.0 GA

Key Features and Updates:

 - Demo changes
   - Added [Stable Video Diffusion](demo/Diffusion)(`SVD`) pipeline.
 - Plugin changes
   - Deprecated Version 1 of [ScatterElements plugin](plugin/scatterElementsPlugin). It is superseded by Version 2, which implements the `IPluginV3` interface.
 - Quickstart guide
   - Updated the [SemanticSegmentation](quickstart/SemanticSegmentation) guide with latest APIs.
 - Parser changes
   - Added support for tensor `axes` inputs for `Slice` node.
   - Updated `ScatterElements` importer to use Version 2 of [ScatterElements plugin](plugin/scatterElementsPlugin), which implements the `IPluginV3` interface.
 - Updated tooling
   - Polygraphy v0.49.13

### CODE REVIEW

**Review by yuanyao-nv — APPROVED** [review]

**Comment by yuanyao-nv** [comment]
Are we missing the onnx submodule update?

**Comment by brb-nv** [comment]
> Are we missing the onnx submodule update?

Right. I'm waiting for the onnxtrt [change](https://github.com/onnx/onnx-tensorrt/pull/987) to merge so that I can get a commit ID to mention here.

**Comment by asfiyab-nvidia** [comment]
Apart from the missing onnx-trt update, rest LGTM

**Review by asfiyab-nvidia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # TensorRT OSS Release Changelog
 
-## 10.2.0 GA - 2024-07-10
+## 10.3.0 GA - 2024-08-02
```
[the remark this answers is not in the document]
Comment (brb-nv) [reply CHANGELOG.md]: Updated. https://github.com/NVIDIA/TensorRT/pull/4062/commits/b09b53892f7b823216281539d510adbd45466638

**Comment by brb-nv** [comment]
> Apart from the missing onnx-trt update, rest LGTM

Updated now. https://github.com/NVIDIA/TensorRT/pull/4062/commits/a264661df257e0e7dac3c6657ba511c705897c85

**Review by kevinch-nv — APPROVED** [review]

**Pull request merged**